addons/payment/views/payment_form_templates.xml

```diff
@@ -408,37 +408,70 @@
 
     <template id="payment.no_pms_available_warning">
         <div class="alert alert-warning mt-2">
-            <strong>No suitable payment option could be found.</strong>
-            <!-- === Availability report button (for debug) === -->
-            <t t-call="payment.availability_report_button"/>
+            <strong>No payment method available for your order.</strong>
+            <!-- === Availability report button === -->
             <br/>
-            If you believe that it is an error, please contact the website administrator.
+            <t t-if="not request.env.is_admin()">
+                <p>If you believe that it is an error, please contact the website administrator.</p>
+            </t>
+            <t t-else="">
```

Comment:
You can swap those condition as checking `True` look better in if instead checking `not True`

Reply:
Done